### PR TITLE
upgrade: Precheck for lbaas version

### PR DESCRIPTION
**Pull request by jsuchome**
We only allow lbaas v2 during the upgrade.

UPDATE: Added check if lbaas config is active. If I read https://docs.openstack.org/admin-guide/networking-adv-features.html#basic-load-balancer-as-a-service-operations correctly, pool is needed for all other kind of objects, so I'm only listing pools in the check ... correct me if I'm wrong

### CODE REVIEW

**Comment by jsuchome** [comment]
[upgrade: Add precheck for lbaasv2](https://trello.com/c/Zyn65Owv/2274-upgrade-add-precheck-for-lbaasv2)

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -835,6 +835,9 @@ en:
         swift_replicas:
           error: 'The number of replicas is bigger than the number of disks assigned for Swift.'
           help: 'Such configuration is not supported in Swift delivered with SOC 7. Adapt your Swift configuration before proceeding with the upgrade.'
+        lbaas_v1:
+          error: 'Old version of LBaaS is configured.'
+          help: 'Such configuration is not supported during the upgrade. Adapt your Neutron configuration to use LBaaS v2 before you continue.'
```
Comment (rhafer) [inline crowbar_framework/config/locales/crowbar/en.yml]: "Such a configuration is no supported for the upgrade. Please switch to LBaaS v2 before you continue"

File: crowbar_framework/app/models/api/crowbar.rb [hunk 2]
```diff
@@ -175,14 +175,23 @@ def openstack_check
         ret = {}
         # swift replicas check vs. number of disks
         prop = Proposal.where(barclamp: "swift").first
-        return ret if prop.nil?
-        replicas = prop["attributes"]["swift"]["replicas"] || 0
+        unless prop.nil?
+          replicas = prop["attributes"]["swift"]["replicas"] || 0
```
Comment (abel-navarro) [inline crowbar_framework/app/models/api/crowbar.rb]: this change, although needed, is unrelated to this PR. Overall looks good to me.

Comment (jsuchome) [inline crowbar_framework/app/models/api/crowbar.rb]: Well, it was not needed until this PR. Do you really think there should be separate PR for that?

Comment (abel-navarro) [inline crowbar_framework/app/models/api/crowbar.rb]: no need for a separate PR although a separate commit would help.

**Comment by jsuchome** [comment]
So I've added some kind of check if lbaas config is active. If I read https://docs.openstack.org/admin-guide/networking-adv-features.html#basic-load-balancer-as-a-service-operations correctly, pool is needed for all other kind of objects, so I'm only listing pools in the check ... correct me if I'm wrong @vuntz @rhafer @abel-navarro
